test/algconvergence/rkc_tests.jl

```diff
@@ -9,7 +9,7 @@ probArr[2] = prob_ode_2Dlinear
 @testset "Power Iteration of Runge-Kutta-Chebyshev Tests" begin
   Random.seed!(123)
   eigen_est = (integrator) -> integrator.eigen_est = 1.5e2
-  for iip in [true, false], Alg in [ROCK2, ROCK4, RKC, SERK2, ESERK4, ESERK5], alg in [Alg(), Alg(eigen_est=eigen_est)]
+  for iip in [true, false], alg in [ROCK4(), ROCK4(eigen_est=eigen_est)]
```

Comment:
They are all testing the same code pass, so I removed most of them.

Reply:
makes sense